cluster/gce/addons/loadbalancing/cloud-provider-role.yaml

```diff
@@ -33,6 +33,13 @@ rules:
   - create
   - patch
   - update
+- apiGroups:
+  - ""
```

Comment:
what does empty mean here?

Reply:
For the coreV1 apis, the apiGroup is left as empty in the existing PolicyRules. I followed the same convention here.